exporters/prometheus/exporter.go

```diff
@@ -233,3 +233,19 @@ func sanitizeRune(r rune) rune {
 	}
 	return '_'
 }
+
+func sanitizeName(n string) string {
+	if len(n) == 0 {
+		return n
+	}
+	var sn = make([]string, len(n))
+	for i, b := range n {
+		if !((b >= 'a' && b <= 'z') || (b >= 'A' && b <= 'Z') || b == '_' || b == ':' || (b >= '0' && b <= '9' && i > 0)) {
```

Comment:
This logic comes from prometheus's [IsValidMetricName](https://github.com/prometheus/common/blob/main/model/metric.go#L92-L102).